<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerrainSpecSectionWidget</class>
 <widget class="QWidget" name="TerrainSpecSectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="xSpinBox">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>y</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="ySpinBox">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Height map URL</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="FileSelectWidget" name="heightmapURLFileSelectWidget"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Mask map URL</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="FileSelectWidget" name="maskMapURLFileSelectWidget"/>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="removeTerrainSectionPushButton">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="FileSelectWidget" name="treeMaskMapURLFileSelectWidget"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Tree mask map URL</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSelectWidget</class>
   <extends>QLineEdit</extends>
   <header>../qt/FileSelectWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
